simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/station/handlers/ocpp/ChangeAvailabilityRequestHandlerTest.java

```diff
@@ -0,0 +1,71 @@
+package com.evbox.everon.ocpp.simulator.station.handlers.ocpp;
+
+import com.evbox.everon.ocpp.simulator.station.handlers.ocpp.support.AvailabilityManager;
+import com.evbox.everon.ocpp.simulator.station.handlers.ocpp.support.AvailabilityStateMapper;
+import com.evbox.everon.ocpp.v20.message.station.ChangeAvailabilityRequest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static com.evbox.everon.ocpp.simulator.station.evse.EvseStatus.AVAILABLE;
+import static com.evbox.everon.ocpp.simulator.support.StationConstants.*;
+import static com.evbox.everon.ocpp.v20.message.station.ChangeAvailabilityRequest.OperationalStatus.OPERATIVE;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(MockitoExtension.class)
+public class ChangeAvailabilityRequestHandlerTest {
+
+
+    @Mock
+    AvailabilityManager availabilityManagerMock;
+    @Mock
+    AvailabilityStateMapper availabilityStateMapperMock;
+
+    @InjectMocks
+    ChangeAvailabilityRequestHandler changeAvailabilityRequestHandler;
+
+    @Test
+    @DisplayName("Should throw an exception on invalid operational status")
```

Comment:
"Should throw IllegalArgumentException on invalid operational status"

Reply:
done